submissions/damisiMahmood/html-forms/Contact Page.html

```diff
@@ -0,0 +1,61 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Form</title>
+    </head>
+    <body>
+        <header><img src="fashionlogo.png" alt="All Things Fashion">
+            <h1> I-DAMSELS ELITE SHOPPING </h1> 
+        </header>
+        <nav>
+            <ul>
+                <li><a href="#"> HOME</a></li> 
+                <li><a href="#">ABOUT</a></li> 
+                <li><a href="#">CATGORIES</a></li> 
+                <li><a href="#">BRANDS</a></li> 
+                <li><a href="#">SIGN OUT</a></li>
+            </ul>
+        </nav>
+        <fieldset>
+            <legend><h1>Contact Us</h1></legend>
+        <form action="#" method="post" enctype="multipart/form-data">
+            <label for="fullname">Full Name</label> <br>
+            <input type="text" name="fullname" id="fullname" placeholder="First name  Last name" size="40" maxlength="30"/><br>
+            <br>
+            <label for="email">Email</label> <br>
+            <input type="email" name="email" id="email" placeholder="Enter your email address"size="40" maxlength="30"/> <br>
+            example@example.com <br>
+            <br>
+            <label for="subject">Subject</label> <br>
+            <input type="text" name="subject" id="Subject" size="40" maxlength="30"/><br> <br>
+            <label for="message">Message</label> <br>
+            <textarea rows="7" cols="50" name="Message" maxlength="300"></textarea>
+            <br>
+            <br>
+
+            <button type="submit">Submit</button>
+        </fieldset>
```

Comment:
The closing tag for the fieldset should not be inside the form element tag. Move it.

Reply:
Done